src/core/config/transformer/subgraph.rs

```diff
@@ -34,6 +34,13 @@ impl Transform for Subgraph {
     type Error = String;
 
     fn transform(&self, mut config: Self::Value) -> Valid<Self::Value, Self::Error> {
+        let enable_federation = config.server.enable_federation;
```

Comment:
```suggestion
        let enable_federation: bool = config.server.get_enable_federation(); 
```

Reply:
Enable only if set to true. Disable by default.

Reply:
done